<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Interface</class>
 <widget class="QMainWindow" name="Interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>815</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>530</width>
    <height>815</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>530</width>
    <height>815</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Interface</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="viewWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>500</width>
      <height>500</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="mazeRadioButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>520</y>
      <width>91</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Maze</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="caveRadioButton">
    <property name="geometry">
     <rect>
      <x>265</x>
      <y>520</y>
      <width>91</width>
      <height>45</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cave</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="solutionStackedWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>570</y>
      <width>291</width>
      <height>211</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="cavePage">
     <widget class="QSpinBox" name="spinBoxCaveBirth">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelCaveBirth">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>20</y>
        <width>58</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Birth</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxCaveDeath">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>50</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>7</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelCaveDeath">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>58</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Death</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushbattonCaveNextGen">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Next generation</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushbattonCaveAutomatic">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>150</y>
        <width>211</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Automatic operation</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxCaveStep">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>190</y>
        <width>61</width>
        <height>22</height>
       </rect>
      </property>
      <property name="minimum">
       <number>100</number>
      </property>
      <property name="maximum">
       <number>1200</number>
      </property>
      <property name="singleStep">
       <number>100</number>
      </property>
      <property name="value">
       <number>700</number>
      </property>
     </widget>
     <widget class="QLabel" name="labelFrequency">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>190</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Frequency</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCaveChance">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>58</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Chance</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBoxCaveChance">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>80</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="mazePage">
     <widget class="QPushButton" name="MazePushButtonViewSolution">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>View solution</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="MazeSpinBoxX1">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>117</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="MazeLabelY2">
      <property name="geometry">
       <rect>
        <x>156</x>
        <y>148</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="MazeSpinBoxY1">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>148</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="MazeSpinBoxX2">
      <property name="geometry">
       <rect>
        <x>177</x>
        <y>117</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="MazeLabelFrom">
      <property name="geometry">
       <rect>
        <x>64</x>
        <y>80</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>From</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="MazeSpinBoxY2">
      <property name="geometry">
       <rect>
        <x>177</x>
        <y>148</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>6</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
     <widget class="QLabel" name="MazeLabelX2">
      <property name="geometry">
       <rect>
        <x>156</x>
        <y>117</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QLabel" name="MazeLabelTo">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>To</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="MazeLabelY1">
      <property name="geometry">
       <rect>
        <x>44</x>
        <y>148</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
     <widget class="QLabel" name="MazeLabelX1">
      <property name="geometry">
       <rect>
        <x>44</x>
        <y>117</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="MazePushButtonClear">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>180</y>
        <width>161</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear solution</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>570</y>
      <width>201</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="settings">
     <item row="0" column="1">
      <widget class="QSpinBox" name="heightSpinbox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>6</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelWidth">
       <property name="text">
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelHeight">
       <property name="text">
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="widthSpinbox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="value">
        <number>6</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>640</y>
      <width>201</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="area">
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonOpen">
       <property name="text">
        <string>Open ready</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButtonSave">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButtonGenerate">
       <property name="text">
        <string>Generate new</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelArea">
       <property name="font">
        <font>
         <pointsize>23</pointsize>
        </font>
       </property>
       <property name="text">
        <string>AREA</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="labelFileCondition">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>790</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
